test/integration.test.ts

```diff
@@ -290,10 +290,14 @@ async function runBundle(
   dir: string,
   args_: string[],
 ): Promise<{ code: number | null; stdout: string; stderr: string }> {
+  const assetPath = process.env.POST_BUILD
+    ? '/../dist/cli.js'
+    : '/../src/cli.ts'
+
   const args = (args_ || []).concat(['--cwd', dir])
   const ps = fork(
     `${require.resolve('tsx/cli')}`,
```

Comment:
the runner should changed to nodejs instead of tsx/cli

Reply:
Maybe use spawn for js.. or?

```ts
const ps = isPostBuild
  ? spawn('node', [__dirname + assetPath].concat(args), {
      stdio: 'pipe',
    })
  : fork(
      `${require.resolve('tsx/cli')}`,
      [__dirname + assetPath].concat(args),
      {
        stdio: 'pipe',
      },
    )
```

Reply:
spawn sounds good 👍

Reply:
I realize it would be similar since the input is js so almost no transform, we could change it later